lte/gateway/c/core/oai/tasks/amf/amf_app_handler.cpp

```diff
@@ -617,13 +616,13 @@ void amf_app_handle_pdu_session_response(
       OAILOG_ERROR(
           LOG_AMF_APP, "pdu session  not found for session_id = %u\n",
           pdu_session_resp->pdu_session_id);
-      return;
+      return rc;
```

Comment:
The assumption here is that `rc` is initialized to RETURNerror and has not been modified since. It is better to be more explicit.
```suggestion
      return RETURNerror;
```

Reply:
addressed the review comment